### PR TITLE
Splitting `other` folder into subfolders

### LINKED ISSUE #688: Break apart the `other` folder into smaller subfolders
The `other` folder has close to 100 policies which means kuttl tests of just that folder take quite some time. Find a strategy and break up those into smaller subfolders allowing tests to run in parallel per folder.

### PR DESCRIPTION
**Pull request by prady0t**
## Related Issue(s)
Closes #688 

## Checklist

- [x] I have read the [policy contribution guidelines](https://github.com/kyverno/policies/blob/main/README.md#contribution).
- [] I have added test manifests and resources covering both positive and negative tests that prove this policy works as intended.
- [] I have added the artifacthub-pkg.yml file and have verified it is complete and correct.

***Note: Same PR as #696***

### CODE REVIEW

**Comment by mintbomb27** [comment]
@prady0t Great work! But as you can see this doesn't bring any change to the time as you need to modify the workflow as well. The `Kuttl Tests` workflow still takes all the tests together from a folder and executes them. 

So would you mind changing the workflow as well to accommodate these changes? Basically, all you need to do I guess is in the `tests` array, instead of just `other`, you'll have to replace it with `other/a-b` and so on based on your folders.

**Comment by prady0t** [comment]
@mintbomb27 Did changes in `test.yml` file. Please have a look.

File: .github/workflows/test.yml [hunk 1]
```diff
@@ -42,7 +42,14 @@ jobs:
...
           - openshift
-          - other
+          # - other
+          - other/a
+          - other/b-d
+          - other/e-l
+          - other/m-p
+          - other/rec-req
+          - other/res
+          - other/s-v
```
Comment (chipzoller) [inline .github/workflows/test.yml:52]: Please rename this folder `s-z` both here in the workflow and the actual folder. Even though there aren't currently policies which begin with letters after `v`, this is where they would go.

**Review by chipzoller — CHANGES_REQUESTED** [review]

**Review by mintbomb27 — APPROVED** [review]
LGTM! Thank you for the contribution!

**Review by chipzoller — APPROVED** [review]

**Comment by chipzoller** [comment]
Thanks, @prady0t! Now we need to make similar adjustments to the workflow in kyverno/kyverno.

**Comment by prady0t** [comment]
I can see folder `other` mentioned in `conformance.yaml` in kyverno/kyverno. Is it supposed to be `other/a` etc. just like the changes we made here? 
Do more such workflow files require such changes?

**Comment by chipzoller** [comment]
That should be the only change.

**Comment by prady0t** [comment]
Ok cool making a PR there them.👍🏻

**Comment by chipzoller** [comment]
Thanks!

**Comment by prady0t** [comment]
@chipzoller Have a look. https://github.com/kyverno/kyverno/pull/7927

**Pull request merged**